app/controllers/planets_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PlanetsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
 
-  def index
-    
+  def show
+    render json: { planets: Planet.all}
   end
-
 end
```

Comment:
this should all be in the index method, not hte show. In addition, I would unnest your json
`render json: Planet.all`